src/main/java/com/volmit/iris/engine/scripting/EngineExecutionEnvironment.java

```diff
@@ -30,6 +31,8 @@
 
     void execute(String script);
 
+    void execute(IrisScript script);
```

Comment:
remove

Reply:
Why would we convert a string to an IrisScript, to a string, to an IrisScript and then run it?